### PR TITLE
feat: poap manager bindings

### PR DESCRIPTION
**Pull request by manu0466**
This PR adds the type definionts for the POAP Manager contract available [here](https://github.com/desmos-labs/desmos-contracts) and provides a set of utility scripts to interact with it.

### CODE REVIEW

**Comment by manu0466** [comment]
This PR  depends on https://github.com/desmos-labs/desmos-contracts/pull/48

**Review by bragaz — APPROVED** [review]
Everything looks good to me. While looking at the schema files I got an idea, what about having a script that update them with the latest version? This might be useful cause it avoid the necessity of doing it by hand each time contracts msgs/queries get updated

**Comment by manu0466** [comment]
> Everything looks good to me. While looking at the schema files I got an idea, what about having a script that update them with the latest version? This might be useful cause it avoid the necessity of doing it by hand each time contracts msgs/queries get updated

@bragaz I can work on that.

File: utils/poap-manager.ts [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        .option("--minter <minter>", "Bech32 address of who will have the minting rights", account!.address)
+        .option("--admin <admin>", "Bech32 address of who will have the contract admin rights", account!.address)
+        .action(async (options) => {
+            console.log(`Initializing contract with code ${options.codeId}`);
+            let instantiateMsg: InstantiateMsg = {
+                admin: options.admin,
+                poap_code_id: options.poapCodeId.toString(),
+                poap_instantiate_msg: {
+                    admin: options.admin,
+                    minter: options.minter,
```
Comment (dadamu) [inline utils/poap-manager.ts:39]: `minter` would be set as the poap manager contract address forcely. I think we can set it as "contract-address" hardcoded string, then remove minter option. What do you think?

File: utils/poap-manager.ts [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                    event_info: {
+                        creator: options.creator,
+                        poap_uri: options.poapUri.toString(),
+                        start_time: options.start,
+                        end_time: options.end,
+                        per_address_limit: options.addressLimit
+                    },
+                    cw721_initiate_msg: {
+                        name: options.name,
+                        minter: options.minter,
```
Comment (dadamu) [inline utils/poap-manager.ts:50]: as above

**Review by dadamu — COMMENTED** [review]
Overall is good. Just I have a minor question want to discuss.

File: utils/poap-manager.ts (hunk 1 above)
Comment (manu0466) [reply utils/poap-manager.ts:39]: Yes totally forgot about that, I remove the `minter` option and put the hardcoded string.

**Review by dadamu — APPROVED** [review]

**Review by bragaz — APPROVED** [review]

**Pull request merged**